lib/marathon.js

```diff
@@ -62,6 +63,21 @@ function Marathon(url, opts) {
                 }
             }
 
+            if (requestStream) {
+                return new Promise((resolve, reject) => {
```

Comment:
Please rewrite to normal function, since this lib is not using latest JavaScript standard. We will introduce Babel or whatever in future.